AmazonChimeSDKDemo/AmazonChimeSDKDemo/VideoTileCell.swift

```diff
@@ -63,7 +63,7 @@ class VideoTileCell: UICollectionViewCell {
         onTileButton.addTarget(self, action: #selector(onTileButtonClicked), for: .touchUpInside)
         onTileButton.isSelected = isVideoPausedByUser
         updateVideoSubscriptionsButton.isHidden = false
-        videoFiltersButton.isHidden = false
+        videoFiltersButton.isHidden = true
```

Comment:
Beside this change, can we hide the option button for local video when ML not imported? The alert controller with a `none` action seems weird.

![Screen Shot 2022-05-04 at 8 51 40 PM](https://user-images.githubusercontent.com/60948810/166861366-bffa6fb3-92bf-440c-a5af-17c09b47e049.png)